### PR TITLE
♻️ Migrate to Maven Central

### PR DESCRIPTION
**Pull request by mishackee**

### CODE REVIEW

**Comment by davidbilik** [comment]
Well i realized one unfortunate thing and I feel bad that you didn't :D `io.github.ackeecz` is not some random thing I came up with. It's based on the github organization of our open source  libraries. But as we've noticed, this is not in the standard ackee organization. The groupId should be `io.github.ackeescreenshoter` and it's needed to request a new app in the sonatype jira 🙃  let's discuss on slack :D

**Comment by mishackee** [comment]
@davidbilik As discussed, changed to the correct `groupId` and new app was created.
a98adbfc7d27416953f530436bd8ec0b63e6d3db

**Comment by davidbilik** [comment]
Let's set the new artifact id to `android` instead of `ass` 😏

**Comment by mishackee** [comment]
There we go 12a722e07827cfae0bf2e2c9381d12d830735810 :)

File: README.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
-[ ![Download](https://api.bintray.com/packages/ackeecz/ass/ass/images/download.svg) ](https://bintray.com/ackeecz/ass/ass/_latestVersion)
-
+[ ![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.github.ackeescreenshoter/ass/badge.svg)](https://maven-badges.herokuapp.com/maven-central/io.github.ackeescreenshoter/ass)
```
Comment (davidbilik) [inline README.md]: replace `ass` with `android`

File: README.md [hunk 2]
```diff
@@ -77,10 +76,10 @@ Ass.setShakeSensitivity(Ass.Sensitivity.Hard)   // makes it harder to activate
 ```
 
 ## Dependency
-Library can be added as a dependency from `jcenter` repository:
+Library can be added as a dependency from `mavenCentral` repository (or the ending `jcenter`):
```
Comment (davidbilik) [inline README.md]: remove jCenter mention, it's not true

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest package="cz.ackee.example.ass"
+<manifest package="io.github.ackeescreenshoter.ass.sample"
```
Comment (davidbilik) [inline app/src/main/AndroidManifest.xml]: hm, can you replace the `ass` in the pkg with `android` as well pls? 😬

**Pull request merged**